### DISCUSSION TITLE
ssh -R Connection

### DISCUSSION THREAD

**Discussion by affkie** [termux/termux-app#3425] [Q&A] [answered]
i want to make a tunnel with this command:
```
ssh -R 80:localhost:80 serveo.net
```

and it only works when mobile hotspot is enabled. why?

sometimes I want to use my wifi but the hotspot turns it off.

is there a way to use tunnel without enabling mobile hotspot?

**Comment by sylirre** [comment, marked as answer]
What DNS provider do you use?

I found that 1.1.1.2 (Cloudflare Security DNS) added Serveo to blacklist (thanks to phish guys). I guess some other DNS providers can be doing same. `1.1.1.1` (Cloudflare but not security enhanced) and `8.8.8.8` (Google) resolve it correctly.

`ssh -R` itself should work without issues as soon as your endpoint host name can be resolved and is reachable. Never had problems with it myself.

**Comment by affkie** [comment]
Thanks!
